docs/release-notes/release-notes-0.6.0.md

```diff
@@ -0,0 +1,149 @@
+# DCRDEX v0.6.0
+
+April 4, 2023
+
+## Important Notices
+
+Always record the "seed" that is generated by the DEX client on initialization (when you set your password the first time in the browser window). This is a string of characters that is used to derive DEX account identities and initialize any native/built-in wallets you create in the DEX client. **You must have this seed to recover funds** that you receive in such wallets if your lose access to your DEX client's data folder.
+
+Although DCRDEX looks and feels like a regular exchange, the decentralized aspect brings an expanded role to the client. Please take the time to read and understand the following:
+
+- If using external wallets, never shut them down while dexc is running.  Always stop dexc before stopping your wallets.
+- If you have to restart dexc with active orders or swaps, you must **immediately login again** with your app password when dexc starts up.
+- There is an "inaction timeout" when it becomes your client's turn to broadcast a transaction, so be sure not to stop dexc or lose connectivity for too long or you risk having your active orders and swaps/matches revoked. If you do have to restart dexc, just remember to login as soon as you start it up again.
```

Comment:
Is this only "when it becomes your client's turn to broadcast a transaction"? I think it starts if the client misses a ping/pong?

Reply:
ping/pong as in the connection?  No, it doesn't matter.  Their booked orders do eventually become unbooked, but as it pertains to active swaps the time disconnected won't cause a problem if they get back before it's too late.